week-07/assessment/field-agent-ui/js/index.js

```diff
@@ -45,16 +94,27 @@ function populateAgents(agents) {
     for (const agent of agents) {
         // TODO: This embedded HTML explicitly attaches a function call for update and delete.
         // Complete the confirmDelete and showUpdate functions.
+
+        //reformatting date
+        const dob = new Date(agent.dob);
+        const newDob = `${dob.getMonth() + 1}/${dob.getDate()}/${dob.getFullYear()}`; //Via Google.
+
+        // Format the height as feet'inches
+        const feet = Math.floor(agent.heightInInches / 12);
+        const inches = agent.heightInInches % 12;
+        const newHeight = `${feet}'${inches}"`;
```

Comment:
Nice new formats.
Have you noticed that some dates are off-by-one when formatted? Try it. I imagine it's because a hard-coded date is GMT and we're flipped a day because of the local offset?